go.sum

```diff
@@ -323,6 +323,8 @@ github.com/zclconf/go-cty v1.12.1/go.mod h1:s9IfD1LK5ccNMSWCVFCE2rJfHiZgi7JijgeW
 github.com/zclconf/go-cty-debug v0.0.0-20191215020915-b22d67c1ba0b/go.mod h1:ZRKQfBXbGkpdV6QMzT3rU1kSTAnfu1dO8dPKjYprgj8=
 gitlab.com/infor-cloud/martian-cloud/tharsis/tharsis-sdk-go v0.9.0 h1:vs9YG89YIj3yu3eaSjipxifOAIRcSpA3PcUWzbV7DeM=
 gitlab.com/infor-cloud/martian-cloud/tharsis/tharsis-sdk-go v0.9.0/go.mod h1:Mo8oWON5Bq1053r1pA+G0CHfTUAEofv2Mq/JlHTFOiE=
```

Comment:
go.sum still shows Tharsis SDK v0.9.0 (as well as 0.10.0).  Was "go mod tidy" run?

Reply:
Nice, catch. Forgot about that. Thanks.